iothub/service/src/PurgeMessageQueueResult.cs

```diff
@@ -21,6 +21,6 @@ public sealed class PurgeMessageQueueResult
         /// The total number of messages that were purged from the device's queue.
         /// </summary>
         [JsonProperty(PropertyName = "totalMessagesPurged", Required = Required.Always)]
-        public int TotalMessagesPurged;
```

Comment:
Is this a breaking change?

Reply:
I don't believe so, public fields are both gettable and settable